core/internal/features/ocr2/features_ocr2_test.go

```diff
@@ -468,7 +471,7 @@ func TestIntegration_OCR2_ForwarderFlow(t *testing.T) {
 		oracles = append(oracles, confighelper2.OracleIdentityExtra{
```

Comment:
nit: can we add a sanity check here that we are indeed going through forwarders (i.e effectiveTransmitter == forwarder || effectiveTransmitter != sendingKeys[0])
iiuc, this test can silently pass (in the future) even though forwarder setup failed

Reply:
I take that back, I see you added a verification in the delegate, kudos